common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -181,21 +181,6 @@ def test_teams_not_enabled_no_topics(self):
         self.set_team_configuration({u"max_team_size": 10, u"topics": []})
         self.verify_teams_present(False)
 
-    def test_teams_not_enabled_not_enrolled(self):
```

Comment:
@dianakhuang: Why this change?

Reply:
Looking at the course home page when not enrolled, no tabs were shown except for the course home tab, so it didn't seem important to have a test for the Teams tab.